### PR TITLE
Update Contributing.md with PR update strategy

### LINKED ISSUE #2372: Add note that we do merge
In #2342 I realized that some people do rebase.
It's not a bad thing but we use merge and we don't want to mix the merging strategies.

In `Contributing.md` add a note that would explain this

### PR DESCRIPTION
**Pull request by zhengow**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

### What's new?

- [x] PR closes #2372 
- [x] Update Contributing.md with PR update strategy

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

### Optional

- [ ] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [ ] I've tested PR on mobile and everything seems works
- [ ] I found edge cases
- [ ] I've written some unit tests 🧪

### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=EgCzZLpVe7zg7RHh79qPyC8UrykVkA22YY2oj7sWMZf15Ws)

### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -186,6 +186,12 @@ For content changes, make sure that you:
 * Check that all of your changes render correctly in staging. Remember, that lists and tables can be tricky.
 * If there are any failing checks in your PR, troubleshoot them until they're all passing.
 
+### Update strategy: always update with `merge commit`
+Try to submit your PR as soon as possible. That's always a good way to avoid conflict with other commits.
+
+However, if it really happens, we want to you update your branch with `merge commit` rather than `rebase`. The reason is
```
Comment (vikiival) [inline CONTRIBUTING.md:192]: always use `merge`

Comment (zhengow) [reply CONTRIBUTING.md]: already change the content, plz check again

**Review by vikiival — COMMENTED** [review]
Legit

**Review by vikiival — APPROVED** [review]

**Comment by vikiival** [comment]
pay 100 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $100 @ 118.12 USD/KSM ~ 0.847 $KSM
🧗 [EgCzZLpVe7zg7RHh79qPyC8UrykVkA22YY2oj7sWMZf15Ws](https://kusama.subscan.io/account/EgCzZLpVe7zg7RHh79qPyC8UrykVkA22YY2oj7sWMZf15Ws)
🔗 [0x74534f3f931c9ebd0179fb3994bbb0cd2d137d8c1e173f1f90eac4deb3272713](https://kusama.subscan.io/extrinsic/0x74534f3f931c9ebd0179fb3994bbb0cd2d137d8c1e173f1f90eac4deb3272713)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**